### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by Manujdixit** [Manujdixit/Memeflick#3]

**Comment by jyotika6221** [comment]
hey is this repo under hacktober, if yes i can contribute.

**Comment by Manujdixit** [comment]
Yes this repository is under hacktoberfest and you can contribute.

**Comment by jyotika6221** [comment]
ok so can you describe what are the updates that need to be done?

**Comment by Manujdixit** [comment]
@jyotika6221 Under this issue you have to update readme.

Link for the document is given below.
[https://docs.google.com/document/d/1OstYxAtI-YASymorDVgC2-1wZIZ2etfE1LbPzwLcqes/edit?usp=sharing](url)

For other issues you can check issues tab under repositories name or you can generate a new issue.

**Comment by jyotika6221** [comment]
ok.
